examples/zkapps/01-hello-world/README.md

```diff
@@ -1,26 +1,39 @@
 # Mina zkApp: 01 Hello World
 
-This template uses TypeScript.
+This Hello World tutorial helps you get started with o1js, zkApps, and programming with zero-knowledge proofs.
 
-## How to build
+## Version:
+- o1js: **0.15.2**
+
+## Tutorial
+
+For step by step tutorial, you can follow this [link](https://docs.minaprotocol.com/zkapps/tutorials/hello-world).
```

Comment:
```suggestion
For the step-by-step tutorial, see [Tutorial 1: Hello World](/zkapps/tutorials/hello-world).
```

Reply:
One of our doc styles is to use descriptive link text (more helpful than "here"). See the [Docs Style Guide](https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide) 
for this tip and other styles you might be interested in.
- [Use short descriptive link text](https://developers.google.com/style/link-text) that helps provide context for the material that you're linking to.

Reply:
Oh Okay. 
I will make the changes in coming commit.

Thank you